api/src/parcel/get.handler.ts

```diff
@@ -51,13 +47,20 @@ async function getParcelData(
   return body;
 }
 
-export const handler = async (event: {
-  pathParameters: ParcelPathParameters;
-}): Promise<APIGatewayProxyResult> => {
-  const coordinates = event.pathParameters?.latlong ?? '';
+export const handler = async (event: APIGatewayProxyEventV2): Promise<APIGatewayProxyResult> => {
+  // Parse out the path parameters.
+  const coordinates = trimPath(event.rawPath)[2];
+  if (!coordinates || coordinates == '')
+    return {
+      statusCode: 400,
+      headers: CORS_HEADERS,
+      body: 'No coordinates provided. Expected {lat},{long}.',
+    };
```

Comment:
Did API Gateway previously handle path validation? Is that why we need to parse and validate in code now?

Reply:
Yes, and this is now obsolete.